.gitignore

```diff
@@ -5,6 +5,7 @@
 
 # production
 /build
+/dist
```

Comment:
new build folder for js code, copied over to dash_table for python packaging

Reply:
I'm pretty sure the Python build goes into `/dist`, when doing `python setup.py sdist`, so not sure if that's a good idea!